CHANGELOG.md

```diff
@@ -1,8 +1,13 @@
+- Changed: use ``react-router`` basename feature
+  ([#165](https://github.com/MoOx/statinamic/pull/165))
```

Comment:
Instead of explaining the technical change here, we should explain what happen for the end user (in this case, all urls in collection do not contain base pathname anymore and all url in components should not have it anymore).